extensions/2.0/Khronos/KHR_materials_transmission/README.md

```diff
@@ -0,0 +1,147 @@
+
+### transmissionFactor 
+
+The amount of light that is transmitted by the surface rather than diffusely re-emitted. This is a percentage of all the light that penetrates a surface (i.e. isn’t specularly reflected) rather than a percentage of the total light that hits a surface. A value of 1.0 means that 100% of the light that penetrates the surface is transmitted through. This is the default.
+
+<figure>
+  <img src="./figures/ConstantTransmission.png"/>
+<figcaption><em>Defaulting to 1.0 allows making a material transparent by simple defining the extension.</em></figcaption>
+</figure>
+
+### transmissionTexture 
+
+A greyscale texture that defines the amount of light that is transmitted by the surface rather than diffusely re-emitted. A value of 1.0 means that 100% of the light that penetrates the surface (i.e. isn’t specularly reflected) is transmitted through. The value is linear and is multiplied by the transmissionFactor to determine the total transmission value. Note that use of transmission values does not disallow usage of alpha coverage (via `baseColor.a`).
+
+<figure>
+  <img src="./figures/TransmissionTexture.png"/>
+<figcaption><em>Controlling transmission amount with transmissionTexture.</em></figcaption>
+</figure>
+
+## Implementing Transmission ##
+
+From the core [glTF BRDF](https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#appendix-b-brdf-implementation), we have:
+
+*f* = *f*<sub>*diffuse*</sub> + *f*<sub>*specular*</sub>
+*f*<sub>*diffuse*</sub> = (1 - *F*) * *diffuse*
+*f*<sub>*specular*</sub> = *F* * *G* * *D* / (4 * dot(*N*, *L*) * dot(*N*, *V*))
+, where *F* is the Surface Reflection Ratio.
+
+We will now add an additional term for the transmitted light:
+
+*f* = *f*<sub>*diffuse*</sub> + *f*<sub>*specular*</sub> + *f*<sub>*transmission*</sub>
+
+*f*<sub>*transmission*</sub> = (1 - *F*) * *T* * *D<sub>T</sub>* * *baseColor*
+where *T* is the transmission percentage defined by this extension's `transmission` and `transmissionTexture` properties and *D<sub>T</sub>* is the distribution function for the transmitted light. The distribution function is the same Trowbridge-Reitz model used by specular reflection except sampled along the view vector rather than the reflection. The *baseColor* factor causes the transmitted light to be tinted by the surface.
+
+Light that penetrates a surface and is transmitted will not be diffusely reflected so we also need to modify the diffuse calculation to account for this.
+*f*<sub>*diffuse*</sub> = (1 - *F*) * (1 - *T*) * *diffuse*
+
+Optical transparency does not require any changes whatsoever to the specular term. So this can all be rearranged so that the transmitted light amounts to a modifaction of the diffuse lobe.
+*f* = (1 - *F*) * (*T* * *D<sub>T</sub>* + (1 - *T*) * *diffuse*) + *f*<sub>*specular*</sub>
+
+
+
```

Comment:
Should there be guidance on how transmission works with IBL?  A hint here would be useful: When dealing with light contributions from an IBL source, your IBL environment map is sampled in the same manner as with the surface reflection, but for transmission we use the view vector rather than the reflected view vector.  

I assume that is how it works.